cluster/saltbase/salt/kube-apiserver/initd

```diff
@@ -38,6 +38,9 @@ DAEMON_USER=kube-apiserver
 #
 do_start()
 {
+        # Raise the file descriptor limit - we expect to open a lot of sockets!
+        ulimit -n 65536
```

Comment:
I copy & pasted this from kube-proxy configuration. I guess it's just because it's 2^16